vm/vmExpr.go

```diff
@@ -256,6 +256,9 @@ func invokeExpr(expr ast.Expr, env *Env) (reflect.Value, error) {
 			return v.FieldByIndex(field.Index), nil
 		case reflect.Map:
 			v = getMapIndex(reflect.ValueOf(e.Name), v)
+			if v == zeroValue {
```

Comment:
Add comment back in?
// Note if the map is of reflect.Value, it will incorrectly return nil when zero value